packages/eas-cli/src/graphql/__tests__/queryTools.test.ts

```diff
@@ -0,0 +1,174 @@
+import { DocumentNode, FragmentDefinitionNode, OperationDefinitionNode } from 'graphql';
```

Comment:
Please rename this file.
The convention is to suffix tests filenames with`-test`, not `.test`.

Reply:
Ok, but I'm gonna miss that colored icon:
![Screenshot 2020-10-22 at 15 43 57](https://user-images.githubusercontent.com/278340/96880594-a2d0ea00-147d-11eb-86ef-22375f17e0c1.png)

Now I have to find a VS Code theme which supports this naming pattern 😅